packages/app-project/src/screens/ClassifyPage/ClassifyPage.js

```diff
@@ -37,16 +56,36 @@ function ClassifyPage (props) {
       >
 
         <Box as='main' fill='horizontal'>
+          {!canClassify && (
+            <Modal
+              active
+              closeFn={onClose}
+              onEsc={onClose}
```

Comment:
I think `closeFn` covers on escape, this might be able to be deleted

Reply:
I think you're right.